lib/matplotlib/tests/test_axes.py

```diff
@@ -6129,6 +6129,14 @@ def test_margins():
                               ymax + (ymax - ymin) * 0.5)
 
 
+def test_margin_getters():
+    fig = plt.figure()
+    ax = fig.add_subplot()
+    ax.margins(0.2)
+    assert ax.get_xmargin() == 0.2
+    assert ax.get_ymargin() == 0.2
```

Comment:
I think it would be better to use different numbers for x and y, so we can verify the right ones end up in the right place.